src/core/python/logger.cpp

```diff
@@ -60,4 +58,6 @@ MI_PY_EXPORT(Logger) {
         .def_method(Logger, read_log);
 
     m.def("Log", &PyLog, "level"_a, "msg"_a);
+    m.def("set_logger", &mitsuba::set_logger, "logger"_a);
```

Comment:
If I remember correctly, `.none()` needs to be added to the argument to make it explicitly allowed?
From the PR it looks like have no logger is allowed, so we could pass None here to do that.